docs/source/articles/testing.html.md

```diff
@@ -794,7 +838,34 @@ Workarea.configure do |config|
 end
 ```
 
-Additionally, you can customize the factory default values for the duration of a single test by utilizing the `Workarea.with_config` method to temporarily modify the configuration values.
+Additionally, you can customize the factory default values for the duration of a single test.
+
+Since Workarea 3.5.0, the global configuration is reset before each test.
+Prior to Workarea 3.5.0, you must wrap temporary configuration changes in `Workarea.with_config` to ensure they reset after the test.
+
+Example for Workarea 3.5 and up:
+
+```ruby
+# test/system/storefront/categories_system_test.decorator
+
+require 'test_helper'
+
+module Workarea
+  decorate Storefront::CategoriesSystemTest do
+
```

Comment:
(minor) (like so minor) I think this will actually fail Rubocop's linting because of the blank line on 855.